### PR TITLE
German language updated

### PR DESCRIPTION
**Pull request by Brambleberry4**
Hopefully improved the German po-file.

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -185,24 +186,24 @@ msgid ""
...
 
 #: ../terminatorlib/container.py:198
 msgid "Do not show this message next time"
-msgstr "Diese Nachricht das nächste Mal nicht anzeigen"
+msgstr "Diese Nachricht in Zukunft nicht mehr anzeigen"
 
 #: ../terminatorlib/encoding.py:34
 msgid "Current Locale"
-msgstr "Derzeitige Standorteinstellungen"
+msgstr "Derzeitiger Standort"
```
Comment (dmaphy) [inline po/de.po:206]: Probably it's more a "Derzeitige Spracheinstellungen"?

File: po/de.po [hunk 2]
```diff
@@ -706,11 +707,11 @@ msgstr "Senkrechter Strich"
 
 #: ../terminatorlib/preferences.glade.h:26
 msgid "GNOME Default"
-msgstr "GNOME-Vorgabe"
+msgstr "GNOME-Standard"
 
 #: ../terminatorlib/preferences.glade.h:27
 msgid "Click to focus"
-msgstr "Klicken zum Fokussieren"
+msgstr "Klicken zum fokussieren"
```
Comment (dmaphy) [inline po/de.po:714]: I'd leave the big F here.

File: po/de.po [hunk 3]
```diff
@@ -914,19 +915,19 @@ msgstr "Titelleiste unten (Neustart erfordelich)"
 
 #: ../terminatorlib/preferences.glade.h:79
 msgid "Hide size from title"
-msgstr "Größe im Titel verstecken"
+msgstr "Größe vom Titel verstecken"
```
Comment (dmaphy) [inline po/de.po:918]: I think "im" is more correct here.

File: po/de.po [hunk 4]
```diff
@@ -1014,7 +1015,7 @@ msgstr "Allgemein"
 
 #: ../terminatorlib/preferences.glade.h:104
 msgid "_Run command as a login shell"
-msgstr "Befehl wie eine Login-Shell _ausführen"
+msgstr "_Befehl als Login-Shell ausführen"
```
Comment (dmaphy) [inline po/de.po:1018]:
```suggestion
msgstr "_Befehl in Login-Shell ausführen"
```

File: po/de.po [hunk 5]
```diff
@@ -103,11 +104,11 @@ msgid ""
 "terminals in grids (tabs is the most common default method, which Terminator "
 "also supports)."
 msgstr ""
-"Werkzeug für erfahrene Nutzer, um Terminals anzuordnen. Es ist inspiriert "
-"von Anwendungen wie: gnome-multi-term, quadkonsole, usw., deren Hauptfokus "
-"auf dem Anordnen der Terminals in einem Gitter liegt (die Verwendung  von "
-"Reitern ist die meist verbreitete Methode, welche von Terminator auch "
-"unterstützt wird)."
+"Ein Werkzeug für erfahrene Nutzer, um Terminals anzuordnen. Es ist von "
+"Anwendungen wie: gnome-multi-term, quadkonsole, usw. inspiriert, deren "
```
Comment (dmaphy) [inline po/de.po:108]:
```suggestion
"Anwendungen wie gnome-multi-term, quadkonsole, usw. inspiriert, deren "
```

File: po/de.po (hunk 1 above)
Comment (Brambleberry4) [reply po/de.po:206]: I am not sure about the correct translation of "Locale"

Comment (Brambleberry4) [reply po/de.po:206]: Do you know about the context?

Comment (dmaphy) [reply po/de.po:206]: Usually the context for the use of the term "locale" is about language or charset encoding. That's why I was suggesting "Spracheinstellungen". An alternative which may be even better would be "(momentaner) Zeichensatz" or something similar.

A quick look into the code told me that this translation is about the context menu when you right click somewhere in the terminal, there is a menu point "Encodings" which is most properly translated as "Zeichensatz" in my opinion.

Comment (Brambleberry4) [reply po/de.po:206]: so I found https://de.wikipedia.org/wiki/Locale where Locale is the umbrella term for all these settings location language, keyboard layout etc.. They translate it with "Einstellungssatz".

Comment (dmaphy) [reply po/de.po:206]: "Einstellungssatz"… yeah, I like that word, even if it sounds a bit unusual to me personally. But seems it's the most proper translation then. (:

Comment (Brambleberry4) [reply po/de.po:206]: Okay I will use that :)

**Review by lazyfrosch — APPROVED** [review]
LGTM

**Comment by lazyfrosch** [comment]
Thanks for your work, I will document the use of Transifex soon, so it will be easier to help translating without the need to work on PO files.

**Pull request merged**